src/circuit/layouter.rs

```diff
@@ -105,6 +104,22 @@ pub trait RegionLayouter<F: Field>: fmt::Debug {
     fn constrain_equal(&mut self, left: Cell, right: Cell) -> Result<(), Error>;
 }
 
+/// Helper trait for implementing a custom [`Layouter`].
+///
+/// This trait is used for implementing table assignments.
+///
+/// [`Layouter`]: super::Layouter
+pub trait TableLayouter<F: Field>: fmt::Debug {
+    /// Assign a fixed value
+    fn assign_fixed<'v>(
```

Comment:
I think we should rename this to `assign_table` or something, as it's restricted only to `TableColumn`.

Reply:
Oh yeah, I was going to rename this to `assign_cell` in a follow-up commit (to keep the `TableColumn` commit simpler) and forgot.